.github/scripts/mqtt_test_v5.py

```
@@ -193,19 +193,21 @@ def test_topic_alias():
-    time.sleep(5)
-    process2.terminate()
-    process2 = subprocess.Popen(pub_cmd,
-                               stdout=subprocess.PIPE,
-                               universal_newlines=True)
-    time.sleep(5)
-    process2.terminate()
-    process2 = subprocess.Popen(pub_cmd,
-                               stdout=subprocess.PIPE,
-                               universal_newlines=True)
+
+    for i in range(5):
```

Ruff (0.15.9)

[warning] Loop control variable `i` not used within loop body

(B007)